runtime/shibuya/src/lib.rs

```diff
@@ -777,6 +799,152 @@ impl pallet_ethereum::Config for Runtime {
     type StateRoot = pallet_ethereum::IntermediateStateRoot<Self>;
 }
 
+parameter_types! {
+    pub CouncilMotionDuration: BlockNumber = 3 * DAYS;
+    pub const CouncilMaxProposals: u32 = 100;
+    pub const CouncilMaxMembers: u32 = 100;
```

Comment:
Perhaps a shorter time is better?

E.g. max 2 days? Or maybe 36 hours?

The max numbers should also be lowered to something realistic.